docs/20_smart-contracts/10_getting-started/10_dune-guide/index.md

```diff
@@ -0,0 +1,634 @@
+
+```shell
+$ dune --create-account bob
+$ dune --create-account alice
+```
+
+Now that your contract account has been created, you can build, deploy, and test your smart contract in the next section.
+
+## Smart Contract Development
+
+DUNE provides at least two commands to create smart contract projects:
+* `create-bare-app`, for simple projects, invoking `cdt` directly
+* `create-cmake-app`, for larger projects, using `cmake` builds
+
+Although a bare smart contract project makes more sense for single contract projects, for this guide you will create a cmake project that you will expand. For larger projects involving multiple contracts and/or web applications, a cmake project makes more sense.
+
+## Hello World Contract
+
+These are the steps that you need to follow to create your "Hello World" smart contract:
+
+[Step 1](#step-1---create-project) - Create new cmake smart contract project `hello`
+[Step 2](#step-2---write-contract) - Write the `hello` smart contract
+[Step 3](#step-3---compile-contract) - Compile the `hello` contract
+[Step 4](#step-4---deploy-contract) - Deploy the `hello` contract
+
+### Step 1 - Create Project
+
+Create the cmake smart contract project `hello` in the current directory by running the following command:
+
+```shell
+dune --create-cmake-app hello .
+```
+
+The command will create the following file structure:
+
+```shell
+ls -l hello
+```
+```
+-rw-r--r-- 1 lupaxel lupaxel  442 Feb 17 10:20 CMakeLists.txt
+-rw-r--r-- 1 lupaxel lupaxel  462 Feb 17 10:20 README.txt
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 build
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 include
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 ricardian
+drwxr-x--- 1 lupaxel lupaxel 4096 Feb 17 10:20 src
+```
+
+```shell
+ls -l hello/src
+```
+```
+-rw-r--r-- 1 lupaxel lupaxel 215 Feb 17 10:20 CMakeLists.txt
+-rw-r--r-- 1 lupaxel lupaxel 110 Feb 17 10:20 hello.cpp
+```
+
+Now you will modify the source file `hello.cpp` to implement your contract.
+
+### Step 2 - Write Contract
+
+Enter the `hello` project main directory:
+
+```shell
+cd hello
+```
+
+Open the `hello` project folder in your code editor or IDE of choice:
+
+```shell
+code .
+```
+
+Open the `include/hello.cpp` C++ source file, type the following, then save the file:
+
+```cpp
+#include <eosio/eosio.hpp>
+
+using namespace eosio;
+
+CONTRACT hello : public contract {
+   public:
+      using contract::contract;
+
+      ACTION world( name user ) {
+         print_f("Hello World from %!\n", user);
+      }
+};
+```
+
+The code above defines the `hello` contract class derived from the `eosio::contract` parent class. Then it defines the `world` action, which simply displays a message using the EOS name argument passed in the `user` parameter. Note the usage of macros CONTRACT and ACTION to simplify the C++ syntax. Also note that we are not implementing the `hello.hpp` C++ header file nor now.
```

Comment:
This is way too much to lay on them in their first tutorial :) That sentence is jam-packed with terms that will make them run for the hills. 

We're already working on a smart contract tutorial so we can just remove this and tell them if they want to learn more about it to check out the smart contracts tutorial.

Reply:
Simplified to a shorter, less technical explanation, as discussed in private.